packages/models/src/models/LivechatContacts.ts

```diff
@@ -331,6 +331,24 @@ export class LivechatContactsRaw extends BaseRaw<ILivechatContact> implements IL
 		);
 	}
 
+	disableByContactId(contactId: string): Promise<UpdateResult | Document> {
+		return this.updateOne(
+			{ _id: contactId },
+			{
+				$set: { unknown: true },
```

Comment:
just like the `disableByVisitorId` I think this should be actually:
```suggestion
				$set: { enabled: false },
```